spec/system/corporate_information_page_spec.rb

```diff
@@ -0,0 +1,127 @@
+RSpec.describe "CorporateInformationPage" do
+  context "when visiting a Corporate Information page" do
+    before do
+      @content_store_response = GovukSchemas::Example.find("corporate_information_page", example_name: "corporate_information_page")
```

Comment:
I'm surprised rubocop has moaned about this, but it normally recommends using a `let` rather than instance variables in tests.